src/main/java/Duke.java

```diff
@@ -9,7 +9,6 @@
     private static int numoftask = 0;
```

Comment:
Perhaps its better to name numOfTask with camel case to comply with the coding standard better?